pkg/templates/charts/toggle/cluster-backup/templates/hub-backup-pod.yaml

```diff
@@ -319,5 +371,12 @@ spec:
                       channel: {{ `{{ $ch }}` }}
                 {{ `{{- end }}` }}
               {{ `{{- end }}` }}
-            {{ `{{- end }}` }}                                                             
+            {{ `{{- end }}` }}
+          customMessage:
+            compliant: |- 
+              This template is compliant. There is no redhat-oadp-operator subscription installation outside of the open-cluster-management-backup namespace,
+              or there is and the version of this operator matches the version of the redhat-oadp-operator subscription installed under the open-cluster-management-backup namespace.
+            noncompliant: |-
+              This template is not compliant. There is a version of redhat-oadp-operator subscription installation outside of the open-cluster-management-backup namespace 
+              and this version doesn't match the the version set by the backup and restore operator Helm chart.
```

Comment:
```suggestion
              This template is compliant because the redhat-oadp-operator subscription is externally installed from the open-cluster-management-backup namespace,
              or the subscription is installed with the namespace and the version of this operator matches the version of the redhat-oadp-operator subscription.
            noncompliant: |-
              This template is not compliant because there is a version of redhat-oadp-operator subscription externally installed from the open-cluster-management-backup namespace 
              and this version doesn't match the the version set by the backup and restore operator Helm chart.
```

Reply:
I made some changes but the requested update is not reflecting the actual message .. so the update looks different, see below :  

          customMessage:
            compliant: |- 
              There is no redhat-oadp-operator subscription installed in a namespace other than the open-cluster-management-backup namespace,
              or a redhat-oadp-operator subscription is installed and matches the version of the redhat-oadp-operator subscription installed under the open-cluster-management-backup namespace.
            noncompliant: |-
              There is a version of redhat-oadp-operator subscription installed in a namespace other than the open-cluster-management-backup namespace 
              and this version doesn't match the version set by the backup and restore operator Helm chart 
              or the version of the redhat-oadp-operator subscription installed under the open-cluster-management-backup namespace.